lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php

```diff
@@ -110,116 +106,68 @@ public static function sortByField(string $name, int $orientation = 1, ?Closure
         };
     }
 
-    public function walkComparison(Comparison $comparison): mixed
+    public function walkComparison(Comparison $comparison): Closure
```

Comment:
I think we should start documenting BC breaks along with PRs, there'll be no easy way by the end of the journey

Reply:
There is a nice upgrade guide already, I'll update it.